docker/btcd/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17.3-alpine as builder
```

Comment:
I guess we forgot to update these ones to `1.18` before?

Reply:
Yeah this is for the dev docker scripts though, so it didn't really impact the release builds or anything like that.